librec/pom.xml

```diff
@@ -21,6 +21,11 @@
             <artifactId>guava</artifactId>
             <version>16.0</version>
         </dependency>
+        <dependency>
```

Comment:
I believe this dependency can be removed.

Reply:
I think so too